share/spice/bible/bible.js

```diff
@@ -14,6 +14,7 @@
             id: 'bible',
             name: 'Answer',
             data: result,
+            signal: 'high',
```

Comment:
We should first try to handle this with the Block Group on the IA Page.

@tagawa recently @bsstoner told me we should't set the signal "high" like this because it can confuse the AnswerBar JS.